### PR TITLE
feat: [Datastore] add sample for GqlQuery

### PR DESCRIPTION
**Pull request by vishwarajanand**
Addresses: https://github.com/GoogleCloudPlatform/php-docs-samples/issues/1683

### CODE REVIEW

File: datastore/api/src/functions/concepts.php [hunk 1]
```diff
@@ -314,16 +315,45 @@ function basic_query(DatastoreClient $datastore)
...
+    AND priority >= @b
+order by
+    priority desc
+EOF,
+        [
+            'bindings' => [
+                'a' => false,
+                'b' => 4,
+            ],
+        ]);
```
Comment (bshaffer) [inline datastore/api/src/functions/concepts.php:340]: Readability: 
```suggestion
    $sql = <<<EOF
SELECT * from Task
where
    done = @a
    AND priority >= @b
order by
    priority desc
EOF;
    $query = $datastore->gqlQuery($sql, [
        'bindings' => [
            'a' => false,
            'b' => 4,
        ],
    ]);
```

File: datastore/api/src/functions/concepts.php [hunk 2]
```diff
@@ -314,16 +315,45 @@ function basic_query(DatastoreClient $datastore)
...
+    // [END datastore_basic_gql_query]
+    return $query;
+}
+
 /**
  * Run a given query.
  *
  * @param DatastoreClient $datastore
```
Comment (bshaffer) [inline datastore/api/src/functions/concepts.php:348]: Add phpdoc 
```suggestion
 * @param DatastoreClient $datastore
 * @param Query|GqlQuery $query
```

File: datastore/api/src/functions/concepts.php [hunk 3]
```diff
@@ -314,16 +315,46 @@ function basic_query(DatastoreClient $datastore)
...
+    AND priority >= @b
+order by
+    priority desc
+EOF;
+    $query = $datastore->gqlQuery($gql, [
+            'bindings' => [
+                'a' => false,
+                'b' => 4,
+            ],
+        ]);
```
Comment (bshaffer) [inline datastore/api/src/functions/concepts.php:340]:
```suggestion
        'bindings' => [
            'a' => false,
            'b' => 4,
        ],
    ]);
```

File: datastore/api/src/functions/concepts.php [hunk 4]
```diff
@@ -314,16 +315,46 @@ function basic_query(DatastoreClient $datastore)
...
+    AND priority >= @b
+order by
+    priority desc
+EOF;
+    $query = $datastore->gqlQuery($gql, [
+                  'bindings' => [
+                      'a' => false,
+                      'b' => 4,
+                  ],
+              ]);
```
Comment (bshaffer) [inline datastore/api/src/functions/concepts.php:340]:
```suggestion
    $query = $datastore->gqlQuery($gql, [
        'bindings' => [
            'a' => false,
            'b' => 4,
        ],
    ]);
```

**Review by bshaffer — APPROVED** [review]

File: datastore/api/src/functions/concepts.php (hunk 4 above)
Comment (vishwarajanand) [reply datastore/api/src/functions/concepts.php]: Oh got it.. thanks for fixing this.

**Pull request merged**